src/app.py

```diff
@@ -23,86 +28,85 @@ def default(self, o):
                 return int(o)
             return super(DecimalEncoder, self).default(o)
 
-def createItem():
-    # Connect to database
+# Helper functions
+def get_all_fortunes():
+    dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
+
+    table = dynamodb.Table('FortuneCookie')
+
+    try:
+        response_obj = table.scan()
+    except ClientError as e:
+        print(e.response['Error']['Message'])
+    else:
+        items = response_obj['Items']
+        return items
+
+def delete_fortune(id):
+    dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
+
+    table = dynamodb.Table('FortuneCookie')
+
+    try:
+        response = table.delete_item(
+            Key={
+                'id' : id
+            }
+        )
+    except ClientError as e:
+        if e.response['Error']['Code'] == "ConditionalCheckFailedException":
+            print(e.response['Error']['Message'])
+        else:
+            raise
+    else:
+        logger.debug("DeleteItem succeeded")
+        logger.debug(json.dumps(get_all_fortunes(), indent=4))
+
+def createItem(fortune_item):
     dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
 
-    # Access database table
     table = dynamodb.Table('FortuneCookie')
 
-    # Make JSON file of fortunes you want to add
-    # rawrawrawr stuff
     id = str(uuid.uuid4())
```

Comment:
id should never be used as an local var, use _id or id_ since `id` is a builtin method

Reply:
__id__

Reply:
_id_

Reply:
Thanks! I changed it.